actionview/lib/action_view/path_set.rb

```diff
@@ -43,7 +43,7 @@ def find(path, prefixes, partial, details, details_key, locals)
     end
 
     def find_all(path, prefixes, partial, details, details_key, locals)
-      search_combinations(prefixes) do |resolver, prefix|
+      search_combinations(*prefixes, path.pluralize) do |resolver, prefix|
```

Comment:
While this passes the suite, this change feels too far removed from the original problem. The resolved directory is coincidentally the path name pluralized. 

I believe resolving the issue will require changes to how the `ObjectRenderer` constructs its set of prefixes to prepend to the ViewPath (depending on the `config.action_view.prefix_partial_path_with_controller_namespace` configuration value).

Reply:
I like where you're going with this. 👍  You may want consider another case in your testing. Your current solution adds a search combination that strips all namespaces, which doesn't seem to cover when a controller-namespaced view would like to render a model from another namespace...

For example, if Scoped::Article had an ActiveStorage `file` attribute:

```ruby
class Scoped::Article < ApplicationRecord
  has_one_attached :file
end
```

The `file` attribute is is namespaced under ActiveStorage, with `to_partial_path` of `"active_storage/attachments/attachment"`.

If we try to render with two partials:

```erb
<%# app/views/scoped/articles/_article.html.erb %>
Rendered controller-namespaced article
<%= render article.file %>

<%# app/views/active_storage/attachments/_attachment.html.erb %>
Rendered namespaced attachment
```

Your current idea removes both "scoped" and "active_storage" namespaces, so the attachment's partial wouldn't be found with:
```diff
- search_combinations(*["scoped/active_storage/attachments", "attachments"])
# => ActionView::MissingTemplate - Missing partial scoped/active_storage/attachments/_attachment
```

Would it make sense to add search paths as each namespace is removed?:

```diff
+ search_combinations(*["scoped/active_storage/attachments", "active_storage/attachments", "attachments"])
```

Then folks aren't forced to pick all-or-nothing namespacing, but can use any combination that suits their needs. Perhaps they'd have an "override" partial in one namespace, but use a general one elsewhere.

Thanks for your work on this!